config/webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+module.exports = {
+    entry: './src/app/app.module.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, './dist')
+    },
+    devServer: {
```

Comment:
FYI as a good practice you should split your specific environment configuration into different files.

Reply:
Already split both files for production and development